reference/configuration/framework.rst

```diff
@@ -1093,8 +1093,61 @@ version_strategy
 
 **type**: ``string`` **default**: ``null``
 
-The service id of the :doc:`asset version strategy </frontend/custom_version_strategy>`
-applied to the assets.
+The service id of the asset version strategy applied to the assets. In addition
+to your own :doc:`custom asset version strategies </frontend/custom_version_strategy>`,
+Symfony defines two services for its :ref:`built-in strategies <component-assets-versioning>`:
+``assets.empty_version_strategy`` and ``assets.static_version_strategy``.
+
+The ``assets.empty_version_strategy`` is useful to override the global asset
+versioning strategy inside an asset package:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+       framework:
+            assets:
+                # this strategy is applied to every asset (including packages) ...
+                version_strategy: 'app.asset.my_versioning_strategy'
+                packages:
+                    foo_package:
+                        # ... except if a package explicitly uses the empty strategy
+                        version_strategy: 'assets.empty_version_strategy'
```

Comment:
if you want to use the empty version strategy, a better solution is to configure the ``version`` as empty rather than overwriting the strategy IMO. It avoids relying on the service id used in FrameworkBundle (se my comment in the symfony issue)